src/app/pages/TopicPage/TopicGrid/index.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useContext } from 'react';
+import styled from '@emotion/styled';
+import { arrayOf, shape } from 'prop-types';
+
+import { ServiceContext } from '#contexts/ServiceContext';
+
+import {
+  GEL_GROUP_2_SCREEN_WIDTH_MAX,
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MAX,
+  GEL_GROUP_4_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+import { C_PEBBLE } from '@bbc/psammead-styles/colours';
+import {
+  GEL_SPACING,
+  GEL_SPACING_HLF_TRPL,
+  GEL_SPACING_DBL,
+  GEL_SPACING_TRPL,
+} from '@bbc/gel-foundations/spacings';
+
+import TopicPromo from '../TopicPromo';
+
+const direction = ({ dir }) => (dir === 'ltr' ? 'right' : 'left');
+const Item = styled.div`
+  vertical-align: top;
+  display: inline-block;
+  margin-bottom: ${GEL_SPACING_DBL};
+  width: 100%;
+  @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+    border-top: 1px ${C_PEBBLE} solid;
```

Comment:
Placeholder styling? If not, we should be using relative values please, not px.

Reply:
yep - this will be built out over multiple PRs - going from nothing to a complete, perfect page would be a huge amount of work for a single PR